post001/src/ofApp.h

```diff
@@ -1,40 +1,27 @@
 #pragma once
 
 #include "ofMain.h"
-#include "sooFramesExporter.h"
-#include "sooParticle.h"
+#include "soo_color_palettes.h"
 
-namespace soo
-{
-struct Properties
-{
-    ofColor color;
-    float radius = 10.f; // constant radius for particles
-    float speed = 3.f; // constant speed for particles
+struct Particle {
+  ofVec2f center_;
+  float radius_ = 10.f;
+  ofVec2f direction_;
+  float speed_ = 3.f;
```

Comment:
add comments for the default values explaining that they are default for the small colored balls moving around